core/src/com/unciv/MainMenuScreen.kt

```diff
@@ -222,6 +238,13 @@ class MainMenuScreen: BaseScreen() {
         }
     }
 
+    private fun openCivilopedia() {
+        val ruleset =RulesetCache[game.settings.lastGameSetup?.gameParameters?.baseRuleset]
+            ?: RulesetCache[BaseRuleset.Civ_V_GnK.fullName]
+            ?: return
```

Comment:
An alternative could be to open a mod selection screen, where you can select a base ruleset mod and extension mods.

Reply:
As was suggested in one of the issues. Honestly, much to do and I think players will get used to it quickly and everybody has their favourite very likely selected in there. Ruleset selection - may be _in_ pedia, hidden when in-game. More important would be to check the pedia map editor loads shows the correct ruleset - don't remember what it does :grimacing: